Source/Model/Message/TextSearchQuery.swift

```diff
@@ -179,7 +183,7 @@ private let zmLog = ZMSLog(tag: "text search")
         self.conversation = conversation
         self.conversationRemoteIdentifier = conversation.remoteIdentifier!
         self.originalQuery = query
-        self.queryStrings = query.normalizedForSearch().components(separatedBy: .whitespacesAndNewlines).filter { $0.count > 1 }
+        self.queryStrings = query.normalizedForSearch().components(separatedBy: .whitespacesAndNewlines).filter { TextSearchQuery.isValid(query: $0) }
```

Comment:
I think it's possible to make it shorter here:

```suggestion
        self.queryStrings = query.normalizedForSearch().components(separatedBy: .whitespacesAndNewlines).filter(TextSearchQuery.isValid)
```